py/bitbox02/bitbox02/bitbox02/bootloader.py

```diff
@@ -75,6 +84,9 @@ def __init__(self, transport: TransportLayer, device_info: DeviceInfo):
             "bb02btc-bootloader": SIGDATA_MAGIC_BTCONLY,
             "bitboxbase-bootloader": SIGDATA_MAGIC_BITBOXBASE_STANDARD,
         }.get(device_info["product_string"])
+        self.version = parse_device_version(device_info["serial_number"])
+        # we remove the prerelease as it messes up same-version comparisons.
```

Comment:
I thought the other comment was clearer :innocent: 

```
        # Delete the prelease part, as it messes with the comparison (e.g. 3.0.0-pre < 3.0.0 is
        # True, but the 3.0.0-pre has already the same API breaking changes like 3.0.0...).
```

Reply:
Copied the same comment here :+1: